docs/dynamic-plugins.md

```diff
@@ -361,6 +482,9 @@ By sequentially adding these plugins and allowing for a deployment restart after
 
 ### Using a custom NPM registry
 
+> [!WARNING]  
+> Using a custom NPM Registry and/or NPMJS is not recommended. A dynamic plugin is a runtime artifact, and that it is not the best usage of an NPM registry to deliver runtime artifacts, especially in a container-based environment such as OCP. Using [Container Registry](#consuming-dynamic-plugins-from-a-container-registry) or [Local Configuration](#local-configuration) is recommended and supported for dynamic plugins production environments.
```

Comment:
Small tweak in the 2nd sentence, take out the `,` and `that` so it reads like "A dynamic plugin is a runtime artifact and it is not the..." (and so on).

Also, should we say "recommended and supported" in that last sentence or maybe just "recommended"?

Reply:
Thanks for your review. I forgot to remove the "supported" part, as discussed before it is a strong statement to have it here.